deploy/basic/entrypoint.sh

```diff
@@ -12,19 +12,10 @@
 export MOCK_MODE=carts,orders,users
 export NODE_ENV=production
 export CATALOGUE_PORT=3005
-export CATALOGUE_URL=http://localhost:3005
-export USERS_URL=http://user
+export CATALOGUE_URL=http://localhost:${CATALOGUE_PORT}
 
 echo "Environment: $(uname -a)";
 
-echo "Checking Installation..."
-if [ ! -d "/app/api/node_modules" ]
```

Comment:
the node_modules is now included on the stack.zip?

Reply:
no. The install is moved to `node.sh`